### PR TITLE
PPF-378 - As an integrator I can edit an existing integration

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Added
- [Add test as an integrator I can edit an existing integration](https://github.com/cultuurnet/publiq-platform/commit/30ba281e3b21a9b98914023ebc9f2a46f8eb59c3)

edit-integration

---
Ticket: https://jira.uitdatabank.be/browse/PPF-378

### CODE REVIEW

File: e2e/tests/integrations/integrator/edit-integration.test.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  await page.getByLabel("Integratienaam").fill(faker.word.adjective());
+  await page.getByLabel("Beschrijving").fill(faker.lorem.lines(2));
+
+
+  await page
+    .locator("div")
+    .filter({ hasText: /^Login URLtestomgevingproductieomgeving$/ })
+    .getByRole("textbox")
+    .first()
+    .fill(faker.internet.url());
```
Comment (simon-debruijn) [inline e2e/tests/integrations/integrator/edit-integration.test.ts:31]: Maybe you could improve the html to use <fieldset> and a <legend> to correctly label grouped inputs. That we can have better selectors

File: e2e/tests/integrations/integrator/edit-integration.test.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  await page
```
Comment (simon-debruijn) [inline e2e/tests/integrations/integrator/edit-integration.test.ts:33]: Same for this one

**Review by simon-debruijn — APPROVED** [review]

File: e2e/tests/integrations/integrator/edit-integration.test.ts (hunk 1 above)
Comment (brampauwelyn) [reply e2e/tests/integrations/integrator/edit-integration.test.ts:31]: I will do it in a seperate PR.
So I can first make to changes in the html and deploy the changes to acc, in that way I can modify the tests locally and see if they succeed.

File: e2e/tests/integrations/integrator/edit-integration.test.ts (hunk 2 above)
Comment (brampauwelyn) [reply e2e/tests/integrations/integrator/edit-integration.test.ts:33]: I will do it in a seperate PR.
So I can first make to changes in the html and deploy the changes to acc, in that way I can modify the tests locally and see if they succeed.

**Review by simon-debruijn — APPROVED** [review]

**Pull request merged**